src/transformers/models/auto/modeling_auto.py

```diff
@@ -446,6 +446,7 @@
     [
         # Model for Object Detection mapping
         ("detr", "DetrForObjectDetection"),
+        ("owlvit", "OwlViTForObjectDetection"),
```

Comment:
This maybe the cause of [https://app.circleci.com/pipelines/github/huggingface/transformers/46856/workflows/0831cfcd-9a88-4420-a868-f131b36754a8/jobs/552799?invite=true#step-111-5713](test failure).

Any suggestions ? Does creating a new  `MODEL_FOR_XX` mapping help?